### PR TITLE
AnalyzeAsaModal:Fix the date range limit on AnalyzeAsaModal  

### PR DESCRIPTION
**Pull request by tobianointing**
The date range picker has been set to the limit of 3 months
on the AnalyzeAsaModal so that user wont be able to pick a
date that is more than 3 months

### CODE REVIEW

File: src/layouts/DefaultLayout.tsx [hunk 1]
```diff
@@ -14,9 +14,8 @@ export function DefaultLayout({ children }: Type) {
 
     return (
         <div
-            className={styles.motherContainer}
             data-theme={theme?.theme ? "dark" : "light"}
-            style={{ backgroundColor: theme?.theme ? "#262626" : "" }}
+            style={{ backgroundColor: theme?.theme ? "#262626" : "#f2fbf6", minHeight: "100vh" }}
```
Comment (OladetounJed) [inline src/layouts/DefaultLayout.tsx:18]: You've override my fix for the white space on larger screens

File: src/layouts/DefaultLayout.tsx [hunk 2]
```diff
@@ -14,9 +14,8 @@ export function DefaultLayout({ children }: Type) {
 
     return (
         <div
-            className={styles.motherContainer}
```
[the remark this answers is not in the document]
Comment (tobianointing) [reply src/layouts/DefaultLayout.tsx:17]: i will revert it

File: src/layouts/DefaultLayout.tsx (hunk 1 above)
Comment (tobianointing) [reply src/layouts/DefaultLayout.tsx:18]: the code is still the same the only difference is that I made the style inline it actually fixed the white spaces
should i still revert?

**Review by OladetounJed — APPROVED** [review]
lgtm

**Pull request merged**